docs/notebooks/tremor/tremor_analysis.ipynb

```diff
@@ -36,23 +38,40 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Preprocessing"
+    "## Preprocessing"
```

Comment:
Deze heeft denk ik een '#' te veel (of de anderen te weinig)

Reply:
Het is hetzelfde als bij de gait notebook toch?